resource-service/src/unit-tests/java/uk/gov/gchq/palisade/service/resource/repository/JpaPersistenceLayerTest.java

```diff
@@ -100,18 +98,18 @@ public void addAndGetReturnsResourceTest() {
         Optional<Stream<LeafResource>> persistenceResponse = persistenceLayer.getResourcesById(resource.getId());
         // Then
         Stream<LeafResource> resourceStream = persistenceResponse.orElseThrow();
```

Comment:
This would be better as:
```
        assertThat(persistenceResponse).isPresent();
        assertThat(persistenceResponse.get()).contains(resource);
```
Better tio do the isPresent here rather than optionals `orElseThrow`, as we get a much nicer error message from AssertJ. Also, AssertJ understands streams. And when reading from a stream it's best to preserve order by using a list.